### PR TITLE
Feature/labx 319/opportunity view bind client info with backend

### PR DESCRIPTION
**Pull request by alanhanaev**

### CODE REVIEW

**Comment by alanhanaev** [comment]
Receiving organisation information will only work after confirming the pool request https://github.com/ChronoBank/LaborX-backend/pull/13

File: src/content/lib/CreateJobContent/CreateJobForm.js [hunk 1]
```diff
@@ -590,5 +589,4 @@ class CreateJobForm extends React.Component {
 
 export default reduxForm({
   form: FORM_CREATE_JOB,
-  validate,
```
Comment (sargonpiraev) [inline src/content/lib/CreateJobContent/CreateJobForm.js:593]: validator is required in this form

File: src/store/opportunity-view/actions.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as backendApi from "../../api/backend"
+import { } from "../user/selectors"
```
Comment (sargonpiraev) [inline src/store/opportunity-view/actions.js:3]: ?

File: src/store/opportunity-view/selectors.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+export const getState = state => state.opportunityView
+
+export const getClientProfile = (profile: ProfileClientModel) => {
+    let clientProfile;
+    if (profile) {
+        clientProfile = profile.submitted || profile.approved || {}
+    }
+    else {
+        clientProfile = {}
+    }
```
Comment (sargonpiraev) [inline src/store/opportunity-view/selectors.js:12]: https://github.com/airbnb/javascript#blocks--cuddled-elses

**Review by sargonpiraev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**